### PR TITLE
Fix: Change title strings

### LINKED ISSUE #1160: Change task titles
# Card

Replace "(No title)" by "Task title"
Don't display "(No content)" when there is no content (display nothing, not even a blank space)

Image

# Panel

Change text from "Task title (optional)" to "Task title"
Don't save tasks without title nor content

Image

### PR DESCRIPTION
**Pull request by sunilk4u**
Partial fix #1160.

Just changed the title strings. Will continue on 2nd once I get updates regarding it.

### CODE REVIEW

File: front/src/modules/activities/timeline/components/TimelineActivity.tsx [hunk 1]
```diff
@@ -166,9 +166,7 @@ export function TimelineActivity({ activity }: OwnProps) {
                 onCompletionChange={completeTask}
               />
               <StyledCardContent>
-                <OverflowingTextWithTooltip
-                  text={body ? body : '(No content)'}
-                />
+                <OverflowingTextWithTooltip text={body ? body : ''} />
```
Comment (charlesBochet) [inline front/src/modules/activities/timeline/components/TimelineActivity.tsx:169]: I would do {body && <OverflowingTextWithTooltip text={body ? body : ''} />}

**Review by charlesBochet — COMMENTED** [review]
Looks good to me for this one, you'll need it to rebase on main or merge main into your branch to resolve conflicts

File: front/src/modules/activities/timeline/components/TimelineActivity.tsx (hunk 1 above)
Comment (sunilk4u) [reply front/src/modules/activities/timeline/components/TimelineActivity.tsx:169]: Oh yeah i just missed it.

**Review by charlesBochet — APPROVED** [review]
LGTM, thank you!

**Pull request merged**